{"variants":[{"paths":["\/documentation\/openusd\/c++\/overlay\/tferrormarkwrapper\/errorsequence\/customstringconvertible-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"article","topicSections":[{"generated":true,"title":"Instance Properties","anchor":"Instance-Properties","identifiers":["doc:\/\/SwiftUsd\/documentation\/OpenUSD\/C++\/Overlay\/TfErrorMarkWrapper\/ErrorSequence\/description"]}],"metadata":{"role":"collectionGroup","title":"CustomStringConvertible Implementations","roleHeading":"API Collection","modules":[{"name":"OpenUSD"}]},"hierarchy":{"paths":[["doc:\/\/SwiftUsd\/documentation\/OpenUSD","doc:\/\/SwiftUsd\/documentation\/SwiftUsd\/WrappedTypes","doc:\/\/SwiftUsd\/documentation\/OpenUSD\/Overlay\/TfErrorMarkWrapper","doc:\/\/SwiftUsd\/documentation\/OpenUSD\/C++\/Overlay\/TfErrorMarkWrapper\/ErrorSequence"]]},"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"identifier":{"url":"doc:\/\/SwiftUsd\/documentation\/OpenUSD\/C++\/Overlay\/TfErrorMarkWrapper\/ErrorSequence\/CustomStringConvertible-Implementations","interfaceLanguage":"swift"},"references":{"doc://SwiftUsd/documentation/OpenUSD/Overlay/TfErrorMarkWrapper":{"navigatorTitle":[{"kind":"identifier","text":"TfErrorMarkWrapper"}],"title":"struct TfErrorMarkWrapper","kind":"symbol","url":"\/documentation\/openusd\/overlay\/tferrormarkwrapper","abstract":[],"type":"topic","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"TfErrorMarkWrapper"}],"identifier":"doc:\/\/SwiftUsd\/documentation\/OpenUSD\/Overlay\/TfErrorMarkWrapper","role":"symbol"},"doc://SwiftUsd/documentation/SwiftUsd/WrappedTypes":{"abstract":[],"role":"collectionGroup","kind":"article","identifier":"doc:\/\/SwiftUsd\/documentation\/SwiftUsd\/WrappedTypes","url":"\/documentation\/swiftusd\/wrappedtypes","title":"Wrapped types","type":"topic"},"doc://SwiftUsd/documentation/OpenUSD":{"kind":"symbol","identifier":"doc:\/\/SwiftUsd\/documentation\/OpenUSD","title":"OpenUSD","type":"topic","url":"\/documentation\/openusd","abstract":[],"role":"collection"},"doc://SwiftUsd/documentation/OpenUSD/C++/Overlay/TfErrorMarkWrapper/ErrorSequence/description":{"kind":"symbol","title":"description","identifier":"doc:\/\/SwiftUsd\/documentation\/OpenUSD\/C++\/Overlay\/TfErrorMarkWrapper\/ErrorSequence\/description","type":"topic","abstract":[],"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"description","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}],"url":"\/documentation\/openusd\/c++\/overlay\/tferrormarkwrapper\/errorsequence\/description"},"doc://SwiftUsd/documentation/OpenUSD/C++/Overlay/TfErrorMarkWrapper/ErrorSequence":{"title":"Overlay.TfErrorMarkWrapper.ErrorSequence","abstract":[{"text":"A sequence of errors caught by a ","type":"text"},{"overridingTitle":"TfErrorMarkWrapper","isActive":true,"identifier":"doc:\/\/SwiftUsd\/documentation\/OpenUSD\/Overlay\/TfErrorMarkWrapper","type":"reference","overridingTitleInlineContent":[{"type":"codeVoice","code":"TfErrorMarkWrapper"}]}],"kind":"symbol","url":"\/documentation\/openusd\/c++\/overlay\/tferrormarkwrapper\/errorsequence","type":"topic","identifier":"doc:\/\/SwiftUsd\/documentation\/OpenUSD\/C++\/Overlay\/TfErrorMarkWrapper\/ErrorSequence","navigatorTitle":[{"text":"ErrorSequence","kind":"identifier"}],"role":"symbol","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ErrorSequence","kind":"identifier"}]}}}